src/common/darktable.c

```diff
@@ -1018,6 +1071,21 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     dt_film_set_folder_status();
   }
 
+  // setup resource fractions now                           def  min small  med large  unrestricted
```

Comment:
I think we may want this table of values to be somehow set in `darktableconfig.xml.in`, but we can do that in a separate PR when all this will settle down.

Reply:
agreed on both comments, will do the second later after settling ...